module/merintr/inventry.c

```diff
@@ -796,7 +796,7 @@ Bool InventoryClick(HWND hwnd, BOOL fDoubleClick, int x, int y, UINT keyFlags)
 Bool InventoryKey(HWND hwnd, UINT key, Bool fDown, int cRepeat, UINT flags)
 {
    ID id;
-   Bool held_down = (flags & 0x4000) ? True : False;  /* Is key being held down? */
+   Bool held_down = (flags & 0x4000) ? true : false;  /* Is key being held down? */
```

Comment:
This change is incorrect.. the allowable values for Bool are True and False.  You can change it to bool if you want.